cpp/box/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# create the target and add the needed properties
+nanobind_add_module(_box SHARED module-box.cc)
+target_link_libraries(_box PUBLIC TBB::tbb)
+# this probably isn't needed for box, but may be needed by other modules
+# target_compile_definitions( box # Avoid deprecation warnings for unsupported
+# NumPy API versions. See #
+# https://numpy.org/doc/1.19/reference/c-api/deprecations.html PRIVATE
+# "NPY_NO_DEPRECATED_API=NPY_1_10_API_VERSION" # Default voro++ verbosity is
+# high. PRIVATE "VOROPP_VERBOSE=1")
+
+if(APPLE)
+  set_target_properties(_box PROPERTIES INSTALL_RPATH "@loader_path")
+else()
+  set_target_properties(_box PROPERTIES INSTALL_RPATH "\$ORIGIN")
+endif()
+
+if(_using_conda OR DEFINED ENV{CIBUILDWHEEL})
+  set_target_properties(_box PROPERTIES INSTALL_RPATH_USE_LINK_PATH True)
+endif()
```

Comment:
```suggestion
```
I don't think this is necessary. Please confirm.

Reply:
Correction, I don't think it is necessary for conda. It may be needed for CIBUILDWHEEL.

Reply:
This is still necessary when I build on linux (WSL2) in a conda environment. Without it, I get a runtime error saying module `_box` cannot find `libtbb.so`.